### PR TITLE
Update podspec for pushing 1.0.0-alpha.1 to trunk

### PR DESCRIPTION
**Pull request by ikesyo**
I will push the podspec to CocoaPods Trunk once this is merged.

### CODE REVIEW

File: ReactiveSwift.podspec [hunk 1]
```diff
@@ -11,11 +11,10 @@ Pod::Spec.new do |s|
   s.author       = "ReactiveCocoa"
   
   s.ios.deployment_target = "8.0"
-  s.osx.deployment_target = "10.10"
+  s.osx.deployment_target = "10.9"
```
Comment (ikesyo) [inline ReactiveSwift.podspec]: To match with the Xcode project.

**Comment by ashfurrow** [comment]
Fantastic work! Was going to do this this morning, glad someone beat me to it :smile:

**Comment by mdiep** [comment]
Sorry I forgot to do this last night. 🙈

**Comment by ashfurrow** [comment]
No worries, feel free to @ me in if I can ever be of help.

**Pull request merged**